app/konnect/runtime-manager/vaults/how-to.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set Up and Use a Vault in Konnect
+content_type: how-to
+---
+
+
+The following example shows you how to replace certificates used in {{site.base_gateway}}
+runtime instances with a reference. You can do the same thing with any [supported fields](/gateway/latest/kong-enterprise/secrets-management/). This is the most basic form of secrets management. 
+
+
+## Set up a vault
+
+First, define your environment variables.
+
+Open the **Vaults** page in {{site.konnect_short_name}}:
+1. From the navigation menu, open {% konnect_icon runtimes %} **Runtime Manager**.
+1. Select a runtime group.
+1. From the expanded runtime group menu, open **Vaults**.
+
+Set up a new vault. For this example, we're going to use the environment variable vault. 
+1. Click **Add vault**.
+1. Choose a vault type. 
+1. Enter an environment variable prefix. This will be the prefix that the vault
+uses to recognize relevant values on the data plane.
+
+    For this example, you can use `MY_SECRET`.
+
+1. Set a prefix that you want to use in references.
+
+    For this example, use `my-secret`.
+
+1. Optionally, add a description for the vault, and tag it.
+
+    Although setting tags is optional, it is recommended. If you ever want to manage
+    your vault configuration declaratively, tagging your vaults and managing subsets of configuration
+    via tags lets you protect your vaults from accidental change or deletion.
+    See the [declarative configuration guide for vaults](/deck/latest/guides/vaults/#best-practices)
+    for more information and best practices.
+
+1. Save the vault configuration.
+
+## Define a reference
+
+Now that you have your environment variables set up, you can define references for them.
+This next step has to be configured on the runtime instance.
+
+For each runtime instance that needs to use this vault,
+define your environment variable and assign a secret value to it. For this example,
+paste in your personal access token as the value:
+
```

Comment:
```suggestion
For each runtime instance that needs to use this vault,
define an environment variable key and assign a secret to it. 

```

Reply:
Paste didn't really make sense here to me because you are doing this in the shell or in a script. The example seems clear without it.

Reply:
Whoops - I originally wrote this topic using a personal access token as the thing to encrypt instead of a cert, but couldn't figure out how to use it in config. This is a leftover.